ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapper.java

```diff
@@ -134,6 +134,8 @@ public Claim from(CCDCase ccdCase) {
             settlement,
             fromNullableUTCtoLocalZone(ccdCase.getSettlementReachedAt()),
             mapSealedClaimDocument(ccdCase.getSealedClaimDocument()),
+            null,
```

Comment:
I will prefer to replace this call with Builder we have provided by lombok. Calling a constructor with very long arguments is not clean

Reply:
The thing I like about a constructor here is that we know all the arguments are called, we want to make sure that we map every field to CCD or not as is our current situation

Reply:
We are not mapping all here as passing null for the time being. Even if we have to map all, does not mean using builder is bad here. 

We are having more than 22 constructor arguments to initialise claim object, IMHO using constructor is more error prone as one can pass arguments in wrong order and have totally incorrect data. calling builder will make sure we are mapping the right fields. It's more readable as well.

Reply:
But when you add more fields it forces you to handle it, making it a compile time error.
Which is a good thing here IMO.

I take your point though, I would be interested in others opinion on this.